velodyne_pointcloud/include/velodyne_pointcloud/point_types.h

```diff
@@ -19,28 +19,39 @@
 #ifndef __VELODYNE_POINTCLOUD_POINT_TYPES_H
 #define __VELODYNE_POINTCLOUD_POINT_TYPES_H
 
-#include <pcl/point_types.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <sensor_msgs/point_cloud2_iterator.h>
 
 namespace velodyne_pointcloud
 {
   /** Euclidean Velodyne coordinate, including intensity and ring number. */
-  struct PointXYZIR
+/*  struct PointXYZIR
```

Comment:
Eliminating the PCL dependency would be nice.

I'll think some more about how to handle the API breakage. Maybe doing that in Kinetic would be allowable, I suppose. The trouble is: most users are still on Indigo, and many of them build it from source.